server/src/main/java/org/candlepin/policy/js/entitlement/EntitlementRules.java

```diff
@@ -638,7 +599,8 @@ private void postUnbindVirtLimit(PoolManager poolManager, Entitlement entitlemen
                     for (int idex = 0; idex < pools.size(); idex++) {
                         Pool derivedPool = pools.get(idex);
                         if (derivedPool.getAttributeValue(Pool.Attributes.DERIVED_POOL) != null) {
-                            poolManager.updatePoolQuantity(derivedPool, virtQuantity);
+                            poolManager.setPoolQuantity(derivedPool, derivedPool.adjustQuantity
```

Comment:
formatting gripe: can we move the whole "derivedPool.adjustQuantity" down so the argument list stays with at least the method name?